iis-2vm-sql-1vm/azuredeploy.json

```diff
@@ -0,0 +1,635 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "envPrefixName": {
```

Comment:
In the storage name, now I’m concatenating uniqueString(resourceGroup().id) which I’m hoping will make the storage name unique.